Cargo.toml

```diff
@@ -29,7 +29,8 @@ crossbeam-channel = "0.3"
 lazy_static = "1.1"
 log = "0.4"
 mio = "0.6"
-rand = "0.5"
+rand = "0.6"
+rand_pcg = "0.1"
```

Comment:
I don't like extra deps, we should make the link container available under a certain flag eventually. Then those deps can be that as well.